### PR TITLE
[web][1/3] Start first batch of auto-generated (already passing) tests for line break

### PR DESCRIPTION
**Pull request by mdebbar**
## Description

This is one of 3 PRs to complete the line break algorithm:

1. **[THIS] Add the first batch of auto-generated (already passing) tests from [here](https://www.unicode.org/Public/13.0.0/ucd/auxiliary/LineBreakTest.txt).**
1. [NEXT] Implement the algorithm and add more test cases.
1. [NEXT] Fix surrogates and add the last batch of test cases.

## Related Issues

https://github.com/flutter/flutter/issues/45932

## Tests

I added the following tests:

- `test/text/line_breaker_test.dart`

### CODE REVIEW

**Comment by mdebbar** [comment]
Please hold off reviewing this PR. I made a mistake and put too many test cases (including ones that are still failing).

File: lib/web_ui/test/text/line_breaker_test_data.dart [hunk 1]
```diff
@@ -0,0 +1,4884 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The list of test cases parsed from:
+/// - https://www.unicode.org/Public/13.0.0/ucd/auxiliary/LineBreakTest.txt
+final List<TestCase> data = _rawData.map(_parse).toList();
+
+final RegExp sp = RegExp(r'\s+');
```
Comment (ferhatb) [inline lib/web_ui/test/text/line_breaker_test_data.dart:9]: suggestion: name space

Comment (mdebbar) [reply lib/web_ui/test/text/line_breaker_test_data.dart]: Done

**Comment by mdebbar** [comment]
Failures are unrelated to this PR. Merging.

**Pull request merged**